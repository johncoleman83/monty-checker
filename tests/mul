#!/bin/bash
#
# check multiplication

montyfile="./bytecodes/mul.m"
correct="./expected/mul.txt"

# send commands
./monty $montyfile > $OUTPUTFILE 2> $ERROROUTPUTFILE &
cat $correct > $EXPECTED

# wait a little bit
$SLEEP $SLEEPSECONDS

# check the result
check_diff

#error
print_error

# clean up
stop_monty
