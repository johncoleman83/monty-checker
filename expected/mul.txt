2
4
3
2
1
0
8
3
2
1
0
